tests/helpers/test_utils.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any, Optional
+
+import jax.numpy as jnp
+
+from ott.geometry import geometry
+from ott.problems.linear import linear_problem
+from ott.solvers.linear import sinkhorn
+
+
+def run_sinkhorn(
```

Comment:
this feels copy pasted from sinkhorn_divergence.

is there a place this could be stored that's more visible? or maybe stored in sinkhorn.py?

Reply:
Added as `sinkhorn.solve` + this function now also supports `rank`, to make it consistent with `gromov_wasserstein.solve`.